lib/views/feedbacks_slack_view.ex

```diff
@@ -1,7 +1,25 @@
 defmodule Aprb.Views.FeedbacksSlackView do
+  import Sentient
+
+  def sentiment_score(message) do
+    Sentient.analyze(message)
+  end
+
+  def sentiment_emoji(score) do
+    case score do
+      score when score >= 2 -> ":simple_smile:"
+      score when score <= -2 -> ":frowning:"
+      _ -> ":neutral_face:"
+    end
+  end
```

Comment:
Agreed, we could add a `SentimentAnalyzerService` that gets text and returns `score` or `emoji` :)